eng/pipelines/coreclr/templates/format-job.yml

```diff
@@ -46,18 +58,8 @@ jobs:
         version: '3.x'
         includePreviewVersions: true
         installationPath: $(Agent.ToolsDirectory)/dotnet
-    - task: UsePythonVersion@0
-      inputs:
-        versionSpec: '3.x' 
-        addToPath: true 
-        architecture: 'x64'
-      condition: ${{ eq(parameters.osGroup, 'windows') }}
-    - task: PythonScript@0
-      displayName: Run tests/scripts/format.py
-      inputs:
-        scriptSource: 'filePath'
-        scriptPath: $(Build.SourcesDirectory)/src/tests/Common/scripts/format.py
-        arguments: '-c $(Build.SourcesDirectory)/src/coreclr -o $(osGroup) -a $(archType)'
+    - script: $(PythonScript) $(Build.SourcesDirectory)/src/tests/Common/scripts/format.py -c $(Build.SourcesDirectory)/src/coreclr -o $(osGroup) -a $(archType)
```

Comment:
Are you guaranteed python is installed in all the images we use?

Reply:
Not sure, but we use the same approach in the rest of our JIT runs that use Python scripts and haven't run into issues yet.